website/docs/gitops-run/get-started.mdx

```diff
@@ -101,7 +102,14 @@ We will start it with `--no-session` as it's a single user
 cluster which we want to use in direct mode. The port-forward points
 at the podinfo pod we will create later.
 
-```
+```bash
+export GITHUB_USER=<your github username>
+
+# you can ignore these two commands if you already created and cloned your repository
+gh repo create podinfo-gitops-run
+git clone git@github.com:$GITHUB_USER/podinfo-gitops-run.git
```

Comment:
If you're using `gh` in the first command, why not in the second? However, I would personally prefer to not distract users by mentioning yet another tool.

```suggestion
git clone git@github.com:$GITHUB_USER/podinfo-gitops-run.git
```